### PR TITLE
Compat docs - Tanya's feedback

### PR DESCRIPTION
**Pull request by merriemcgaw**
Tanya had some great feedback for me to incorporate. Adding a review request just to get one more pass over this before I call it done.

### CODE REVIEW

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ __How to opt in or out of these changes__
   
 In order for the application to benefit from these changes, it must run on the .NET Framework 4.7.1 or later. The application can benefit from these changes in either of the following ways:
 - It is recompiled to target the .NET Framework 4.7.1. These accessibility changes are enabled by default on Windows Forms applications that target the .NET Framework 4.7.1 or later.
-- It opts out of the legacy accessibility behaviors by adding the following [AppContext Switch](https://docs.microsoft.com/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) to the ```<runtime>``` section of the app config file and setting it to false, as the following example shows.
+- It opts out of the legacy accessibility behaviors by adding the following [AppContext Switch](https://docs.microsoft.com/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) to the ```<runtime>``` section of the app config.file and setting it to false, as the following example shows.
```
Comment (rpetrusha) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: app config.file --> app.config file

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 2]
```diff
@@ -55,28 +55,28 @@ The .NET Framework 4.7.1 adds support for the fillowing UI automation patterns a
...
 - The `T:System.Windows.Forms.DataGridViewCheckBoxCell` control supports the [Toggle Pattern](https://docs.microsoft.com/dotnet/api/system.windows.automation.togglepattern)
-- The `T:System.Windows.Forms.NumericUpDown` and `T:System.Windows.Forms.DomainUpDown` controls support the [Name](https://docs.microsoft.com/dotnet/api/system.windows.automation.automationelement.nameproperty) and have a [ControlType](https://docs.microsoft.com/dotnet/framework/ui-automation/ui-automation-support-for-the-spinner-control-type) of <xref:System.Windows.Automation.ControlType.Spinner?displayProperty=fullName> and have support for the 
+- The `T:System.Windows.Forms.NumericUpDown` and `T:System.Windows.Forms.DomainUpDown` controls support the [Name](https://docs.microsoft.com/dotnet/api/system.windows.automation.automationelement.nameproperty) and have a [ControlType](https://docs.microsoft.com/dotnet/framework/ui-automation/ui-automation-support-for-the-spinner-control-type) of <xref:System.Windows.Automation.ControlType.Spinner?displayProperty=fullName>.
```
Comment (rpetrusha) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: possibly add "automation element" after name?

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 3]
```diff
@@ -55,28 +55,28 @@ The .NET Framework 4.7.1 adds support for the fillowing UI automation patterns a
...
-- Improved keyboard navigation between the grid and the toolbar above it. Pressing "Shift-Tab" now selects the first ToolBar button, instead of the whole ToolBar
-- `T:System.Windows.Forms.PropertyGrid` controls displayed in High Contrast mode will now draw a focus rectangle around the toolstrip button which corresponds to the current `P:System.Windows.Forms.PropertySort` property value.
+- The `T:System.Windows.Forms.PropertyGrid` control rows which represent header items when the `T:System.Windows.Forms.PropertyGrid` control has a `P:System.Windows.Forms.PropertySort` property set to `F:System.Windows.Forms.PropertySory.Categorized` have a [ControlType](https://docs.microsoft.com/dotnet/framework/ui-automation/ui-automation-support-for-the-menubar-control-type) property value <xref:System.Windows.Automation.ControlType.Button?displayProperty=fullName>
```
Comment (rpetrusha) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: property value --> property value of

File: Documentation/compatibility/winforms-accessibility-changes-471.md [hunk 4]
```diff
@@ -55,28 +55,28 @@ The .NET Framework 4.7.1 adds support for the fillowing UI automation patterns a
...
+- `T:System.Windows.Forms.Button`, `T:System.Windows.Forms.CheckBox`, `T:System.Windows.Forms.RadioButton`, `T:System.Windows.Forms.Label`, `T:System.Windows.Forms.LinkLabel` and `T:System.Windows.Forms.GroupBox` with`P:System.Windows.Forms.Control.Enabled` set to _false_, used a shaded color to render text in High Contrast themes, resulting in low contrast against the background. Now these controls use "Disabled Text" color defined by the OS. This fix applies to controls with `P:System.Windows.Forms.Control.FlatStyle` property set to a value other than `F:System.Windows.Forms.FlatStyle.System`. The latter controls are rendered by the OS.
```
Comment (rpetrusha) [inline Documentation/compatibility/winforms-accessibility-changes-471.md]: applies to controls with --> applies to controls with their 
`F:System.Windows.Forms.FlatStyle.System` is best as \<xref:System.Windows.Forms.FlatStyle.System?displayProperty=nameWithType> so the type name is included.

File: Documentation/compatibility/winforms-accessibility-changes-471.md (hunk 2 above)
Comment (merriemcgaw) [reply Documentation/compatibility/winforms-accessibility-changes-471.md]: I don't think that's quite right. I'm wanting to say that they have a UIA controltype of "Spinner Control". Perhaps just add "control". ?

Comment (merriemcgaw) [reply Documentation/compatibility/winforms-accessibility-changes-471.md]: On second thought, I'd prefer to leave this as is.

Comment (Tanya-Solyanik) [reply Documentation/compatibility/winforms-accessibility-changes-471.md]: ... " and have a UIA ControlType property of "...

Comment (merriemcgaw) [reply Documentation/compatibility/winforms-accessibility-changes-471.md]: If we do that for one instance of ControlType property then we'd want to do it for all. What do you think @rpetrusha ?

**Comment by rpetrusha** [comment]
This looks good, @merriemcgaw. Is it ready to merge?

**Pull request merged**